app/Models/DatabaseDAO.php

```diff
@@ -164,4 +163,168 @@ public function ensureCaseInsensitiveGuids() {
 	public function minorDbMaintenance() {
 		$this->ensureCaseInsensitiveGuids();
 	}
+
+	const SQLITE_EXPORT = 1;
+	const SQLITE_IMPORT = 2;
+
+	public function dbCopy($filename, $mode, $clearFirst = false) {
+		$error = '';
+
+		$userDAO = FreshRSS_Factory::createUserDao();
+		$catDAO = FreshRSS_Factory::createCategoryDao();
+		$feedDAO = FreshRSS_Factory::createFeedDao();
+		$entryDAO = FreshRSS_Factory::createEntryDao();
+		$tagDAO = FreshRSS_Factory::createTagDao();
+
+		switch ($mode) {
+			case self::SQLITE_EXPORT:
+				if (@filesize($filename) > 0) {
+					$error = 'Error: SQLite export file already exists: ' . $filename;
+				}
+				break;
+			case self::SQLITE_IMPORT:
+				if (!is_readable($filename)) {
+					$error = 'Error: SQLite import file is not readable: ' . $filename;
+				} elseif ($clearFirst) {
+					$userDAO->deleteUser();
+					if ($this->bd->dbType() === 'sqlite') {
```

Comment:
(From mobile) because SQLite is the only one to have a database-level VACUUM while MySQL and PostgreSQL do it at table label (which we cannot apply because they have just been deleted). I will add a comment.